android/cli/commands/_build.js

```diff
@@ -2184,6 +2184,7 @@ AndroidBuilder.prototype.generateRootProjectFiles = async function generateRootP
 	const gradleProperties = await gradlew.fetchDefaultGradleProperties();
 	gradleProperties.push({ key: 'android.useAndroidX', value: 'true' });
 	gradleProperties.push({ key: 'android.enableJetifier', value: 'true' });
+	gradleProperties.push({ key: 'android.suppressUnsupportedCompileSdk', value: '33' });
```

Comment:
This line was not changed in the 11_1_X branch, is it necessary?

Reply:
only to target SDK 33. Otherwise it will show a warning all the time. 11_1_X is not targeting 33 so I didn't include it there